notes/force-move-spec.md

```diff
@@ -0,0 +1,208 @@
+# Force Move Spec
+
+To serve as a succinct specification of the ForceMove protocol.
+Everthing required for getting to an outcome.
+
+Things have changed since the original paper.
+Names have changed. Optimizations have been added.
+This should serve as the definitive version over time.
+
+Not a concrete specification.
+Abstract notion of "the state of the channel".
+Don't specify formats.
+In methods, we provide the minimal required parameters.
+In a concrete implementation, the parameters might change - for example it can be more efficient
+
+## Off-chain state
+
+| **Field**      | **Data type**                 | **Definition / Explanation**                                                                         |
+| :------------- | :---------------------------- | :--------------------------------------------------------------------------------------------------- |
+| ChainID        | `bytes32`                     | e.g. ropsten / mainnet                                                                               |
+| Participants   | `address[]`                   | addresses of participants. Determines keys used to sign updates                                      |
+| ChannelNonce   | `uint256`                     | unique, to prevent replay attacks                                                                    |
+| TurnNum        | `uint256`                     | turn number                                                                                          |
```

Comment:
Recommend renaming to `VersionNumber`

Reply:
Let's discuss this. I think there's some merit to the `turnNum` being the thing that determines whose turn it is, but `versionNumber` also has upsides.